data/moves.ts

```diff
@@ -15383,11 +15375,11 @@ export const Moves: {[moveid: string]: MoveData} = {
 		priority: 0,
 		flags: {protect: 1, mirror: 1, authentic: 1, mystery: 1},
 		onTryHit(target, source) {
-			const bannedAbilities = [
-				'asoneglastrier', 'asonespectrier', 'battlebond', 'comatose', 'disguise', 'gulpmissile', 'hungerswitch', 'iceface', 'illusion', 'multitype', 'neutralizinggas', 'powerconstruct', 'rkssystem', 'schooling', 'shieldsdown', 'stancechange', 'wonderguard', 'zenmode',
-			];
+			const additionalBannedAbilities = ['hungerswitch', 'illusion', 'neutralizinggas', 'wonderguard'];
```

Comment:
Huh, so Flower Gift, Forecast, Imposter, Power of Alchemy, Receiver and Trace can be Skill Swapped?

Reply:
Yep, that's right. I thought Imposter in particular was weird that it could be Skill Swapped, but it makes sense (say you send out Imposter into a Substitute or something). Skill Swap uses the same list for both the source and the target, which like I have here is just the general bans and Illusion + Wonder Guard (though behavior in Gen 8 indicates that NGas and Hunger Switch also fall in this group). Ain't it bizarre? Skill Swap is kind of out there on its own.